### DISCUSSION TITLE
Feed filter Follows icon vs Recommendations icon

### DISCUSSION THREAD

**Discussion by ap--** [community/community#19591] [Other Feature Feedback, Questions, & Ideas]
The feed sometimes suggests to follow new users, and the icon for those suggestions (in the Feed) is identical to the icon used in the filter settings for "Follows".

To disable suggestions for users to follow, you have to disable the "Recommendations" setting.
This is a bit confusing and unintuitive.

I would expect to either make the different recommendations distinct, or to make recommendations for different types configurable independently.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by ap--** [reply]
@jdevstatic thank you for the message. When this issue was first created ~1month ago, the functionality you describe as new already existed. This issue is specifically about a the `Recommendations` settings in the filter configuration. Cheers, Andreas 😃
